stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -43,6 +45,7 @@ spec:
             name: openvpn
             readOnly: false
           - mountPath: /etc/openvpn/certs
+            subPath: {{ .Values.persistence.subPath }}
```

Comment:
Why not make this optional instead of having it always use the subPath? I understand it won't actually break anything, just wondering if it isn't cleaner to simply not set it if a user does not require it.

Reply:
ok for me 👍